backend/build.gradle

```diff
@@ -47,6 +47,7 @@ dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
     compile "org.springframework.boot:spring-boot-starter-web:$spring_boot_version"
     compile "org.springframework.boot:spring-boot-starter-security:$spring_boot_version"
+    compile "org.springframework.security:spring-security-acl:5.1.4.RELEASE"
```

Comment:
I think we have another Spring Security dependency. If they use the same versions, can we maybe have a constant for that?